src/api/transferAndWithdrawUsingVaultId.js

```diff
@@ -0,0 +1,9 @@
+const { post } = require('request-promise')
+
+module.exports = async (dvf, transferData) => {
+  const url = dvf.config.api + '/v2/trading/w/transferAndWithdraw'
```

Comment:
Probably `/v1/trading/w/...` ?

Reply:
hmm, this worked in local UI, but it does seem like it should be v1/trading

Reply:
that was actually an unused file, so removed. nice catch